docs/sources/datasources/elasticsearch.md

```diff
@@ -103,6 +109,15 @@ Each data link configuration consists of:
 The Elasticsearch query editor allows you to select multiple metrics and group by multiple terms or filters. Use the plus and minus icons to the right to add/remove
 metrics or group by clauses. Some metrics and group by clauses haves options, click the option text to expand the row to view and edit metric or group by options.
 
+## PPL Query editor
```

Comment:
```suggestion
## PPL query editor
```

Reply:
Thank you for the review! All the edits should be addressed now. 
Quick question, I capitalized the PPL Query editor as it was done so in the above section "Metric Query editor". Please let me know if you would like that one to be also uncapitalized.

Reply:
If it is in the UI, then match the UI.

Otherwise, all headings should be title case, so uncapitalize both. (Sorry, I am not super familiar with Elastic or the UI, so if I am wrong and it is in the UI, let me know.)

Reply:
That makes sense, thank you! PPL query editor is in the UI.